csharp_infer_wrapper/cpp_full_wrapper/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.0.0)
+
+project(infer VERSION 1.0.1 DESCRIPTION "yolov5 inference on hailo-8")
+
+set(COMPILE_OPTIONS -Wall -std=c++20 -Werror -O3)
+set(CMAKE_CXX_STANDARD 20)
+
+add_library(infer SHARED
+    double_buffer.hpp
+    common.h
+    yolo_post_processing.hpp
+    yolo_post_processing.cpp
+    infer.cpp)
```

Comment:
If this is for compilation then the header files and cpp files should be seperated no?